dll/cpl/sysdm/lang/bg-BG.rc

```diff
@@ -8,8 +8,7 @@ BEGIN
     CONTROL "", IDC_ROSIMG, "Static", SS_OWNERDRAW, 4, 5, 248, 111
     LTEXT "Уредба:", IDC_STATIC, 4, 132, 32, 9
     LTEXT REACTOS_STR_PRODUCT_NAME, IDC_STATIC, 10, 144, 116, 9
-    LTEXT "Версия:", IDC_STATIC, 10, 153, 40, 9
-    LTEXT REACTOS_STR_PRODUCT_VERSION, IDC_STATIC, 45, 153, 91, 9
+    LTEXT "Версия: %s", IDC_ROSVERSION, 10, 153, 131, 9
```

Comment:
Do you foresee a need to change this format for any language? If not, might as well just do a string concatenation in the code rather than a printf.